tests/php/FormField/UserFormsOptionSetFieldTest.php

```diff
@@ -17,7 +17,7 @@ public function testCustomErrorMessageValidationAttributesHTML()
         $radio->Required = true;
         $radio->CustomErrorMessage = 'My custom error message with \'single\' and "double" quotes';
         $userFormsOptionSetField = $radio->getFormField();
-        $html = $userFormsOptionSetField->renderWith(UserFormsOptionSetField::class)->getValue();
+        $html = $userFormsOptionSetField->setTemplate(UserFormsOptionSetField::class)->Field()->getValue();
```

Comment:
The `Options` used in the template is actually an extra property that gets added in the `Field()` method.
Before this silently failed because it ended up looping over the "iterator" provided by `ViewableData::getIterator()` in an empty `DBText`.

Doesn't ultimately affect what the test is meant to test, so no need to backport this.